<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogICCAD2017</class>
 <widget class="QDialog" name="DialogICCAD2017">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cells_choose">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose...</string>
   </property>
  </widget>
  <widget class="QLabel" name="cells_lef">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>8</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Cells Lef:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="placed_def">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Placed def:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cells_lef_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>331</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tech_lef_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>45</y>
     <width>331</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="tech_choose">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>45</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose...</string>
   </property>
  </widget>
  <widget class="QLabel" name="erro">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#cc0000;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="tech_lef">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>45</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Tech Lef:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="placed_def_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>331</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="placed_choose">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>80</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
